sdk/build.gradle

```diff
@@ -75,19 +75,18 @@ android {
     sourceSets { main { assets.srcDirs = ['src/main/assets', 'src/androidTest/assets/'] } }
 }
 
-publish {
-    def groupProjectID = 'com.karhoo'
-    def artifactProjectID = 'nsdk-canary'
-    def publishVersionID = '1.2.2.18032021-1'
-    def repoNameID = 'Android'
-
-    userOrg = 'flit-tech'
-    repoName = repoNameID
-    groupId = groupProjectID
-    artifactId = artifactProjectID
-    publishVersion = publishVersionID
-    desc = 'Karhoo Network SDK'
-    website = 'https://developer.karhoo.com'
+afterEvaluate {
+    publishing {
```

Comment:
I did a quick investigation and it seems Maven-publish does not support adding metadata for description/website etc 
Did you come across any documentation on an extended supported environmental variable?

Reply:
Is that a requirement for the solution?

Reply:
it is not a must, I was trying to point out what we may lose